yaus_c4gt_frontend/src/pages/BulkLink.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState, useRef } from "react";
+import axios from 'axios';
+import * as XLSX from "xlsx";
+import { Table } from "antd";
+import { nanoid } from 'nanoid'
+import { Excel } from "antd-table-saveas-excel";
+import { Form, Steps, Input, Select, Tooltip, Button, Space, Layout } from "antd";
+
+
+
+function Bulk() {
+
+    const [form] = Form.useForm();
+    const baseUrl = " https://yaus.xyz";
+    const [data, setdata] = useState([])
+    const [excelData, setExcelData] = useState(null);
+    const len = 0;
+    const [state, setState] = useState({
+        userID: '0fe6ff38-fc46-11ec-b939-0242ac120001',
+        url: '',
+        project: '0fe6ff38-fc46-11ec-b939-0242ac120002',
+        customHashId: '',
+    });
+     
+
+    const columns = [
```

Comment:
Shift it to constant file.

Reply:
resolved in 80396e97b161ddd30d0022b950c212eb1adce878